src/libraries/System.Runtime/tests/System/Reflection/NullabilityInfoContextTests.cs

```diff
@@ -969,14 +1003,14 @@ internal class GenericTest<T>
         public Tuple<T?, string?, string?>? PropertyTupleNullNullNullNull { get; set; }
         public Tuple<T, T?, string> PropertyTupleNonNullNonNon { get; set; } = null!;
         Tuple<string?, T, T?>? PropertyTupleNullNonNullNull { get; set; }
-        public Tuple<T, string?, string>? PropertyTupleNonNullNonNull { get; set; }
-        public Tuple<string, string, T> PropertyTupleNonNonNonNon { get; set; } = null!;
+        public Tuple<T, int?, string>? PropertyTupleNonNullNonNull { get; set; }
```

Comment:
was this coverage of `Tuple<T, string?, string>?` etc worth keeping as well?

Reply:
While fixing this bug found that when a value type was used as a type parameter the nullability of that parameter is omitted which is causing another bug. 

For example for `Tuple<T, string?, string>?` the nullability byte array will look like [2,1,2,1], but for `Tuple<T, int?, string>?` it would become [2,1,1]. So this update for testing that bug